### PR TITLE
ZJ1.7_branch

### PR DESCRIPTION
**Pull request by michaljedro**

### CODE REVIEW

File: app/scss/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        margin-right: auto;
+        max-width: 180px;
+        text-align: left;
+    }
+    &__item-box {
+        display: flex;
+    }
+    &__item-button {
+        max-width: 25%;
+        min-width: 25%;
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:22]: Why not just `width: 25%`?

File: app/scss/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        height: 20px;
+        border: none;
+        background-color: #f09713;
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:25]: Use SCSS variable here.

File: app/scss/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+.copyright {
+    background-color: #464646;
+    text-align: center;
+    padding: 10px;
+    &__text {
+        color: white;
+    }
+    &__text-modify {
+        color: #dc6121;
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:61]: SCSS variable.

File: app/scss/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    }
+}
+
+.bgc_1 {
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:65]: `bgc_1` doesn't say much about this class. Can you find a better name?

File: app/scss/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    &__item-sign:nth-of-type(3) {
+        margin-left: 22px;
+    }
+    &__item-ulcontainer {
+        position: relative;
+        right: -11%;
+    }
+}
+
+.copyright {
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:53]: This section also has `sans-serif` font.

File: app/scss/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        color: white;
+        font-size: 10px;
+        outline: none;
+    }
+    &__item-input {
+        max-width: 75%;
+        height: 20px;
+        outline: none;
+    }
+    &__item-h3 {
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:35]: Putting tag name in class name is tricky - when you change the tag, the name will be confusing. You may just use `item-header`.

File: app/scss/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    }
+    &__item-sign:nth-of-type(2) {
+        margin-left: 22px;
+    }
+    &__item-sign:nth-of-type(3) {
+        margin-left: 22px;
+    }
+    &__item-ulcontainer {
+        position: relative;
+        right: -11%;
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:49]: Try to find another way to position the list.

File: app/scss/_footer.scss [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+        margin-bottom: 15px;
+    }
+    &__item-sign:nth-of-type(1) {
+        margin-left: 5px;
+    }
+    &__item-sign:nth-of-type(2) {
+        margin-left: 22px;
+    }
+    &__item-sign:nth-of-type(3) {
+        margin-left: 22px;
```
Comment (pawel-schmidt) [inline app/scss/_footer.scss:45]: What would you do if the client asks about adding another icon? Will you add `&__item-sign:nth-of-type(4)` selector? 🤔

File: index.html [hunk 9]
```diff
@@ -74,7 +74,46 @@ <h1>Lorem ipsum dolor sit amet.</h1>
...
+        <div class="footer__item-signs">
+          <i class="fab fa-facebook-square fa-2x footer__item-sign"></i>
+          <i class="fab fa-twitter-square fa-2x footer__item-sign"></i>
+          <i class="fab fa-instagram fa-2x footer__item-sign"></i>
+        </div>
+      </div>
+    </div>
+    <div class="copyright">
+      <h3 class="copyright__text">Copyright
+        <span class="copyright__text-modify"> Zaplanujjedzonko.pl</span>
```
Comment (pawel-schmidt) [inline index.html:113]: Redundant space before `Zaplanuj` word.

File: index.html [hunk 10]
```diff
@@ -74,7 +74,46 @@ <h1>Lorem ipsum dolor sit amet.</h1>
...
+    <div class="footer">
+      <div class="footer__item">
+        <h3 class="footer__item-h3">Lorem ipsum dolor</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aspernatur velit reiciendis eligendi ratione fugiat?
+          Architecto recusandae rem laudantium odio.
+        </p>
+      </div>
+      <div class="footer__item">
+        <h3 class="footer__item-h3">Lorem ipsum dolor</h3>
+        <div class="footer__item-ulcontainer">
```
Comment (pawel-schmidt) [inline index.html:87]: Is this container actually necessary here?

File: index.html [hunk 11]
```diff
@@ -74,7 +74,46 @@ <h1>Lorem ipsum dolor sit amet.</h1>
...
+        </div>
+      </div>
+      <div class="footer__item">
+        <h3 class="footer__item-h3">Lorem ipsum dolor</h3>
+        <div class="footer__item-box">
+          <input class="footer__item-input" type="text">
+          <button class="footer__item-button" type="submit" value="submit">Lorem
+          </button>
+        </div>
+        <br>
```
Comment (pawel-schmidt) [inline index.html:103]: `<br>` tag should be used only in text tags like `p`, `h1`, `h2`, … - please add proper styles to add a space between sections.

File: index.html [hunk 12]
```diff
@@ -74,7 +74,46 @@ <h1>Lorem ipsum dolor sit amet.</h1>
       </div>
     </div>
   </div>
-
+  <footer class="bgc_1">
+    <div class="footer">
```
Comment (pawel-schmidt) [inline index.html:78]: Why do not add `footer` class to `footer` element?

**Review by pawel-schmidt — CHANGES_REQUESTED** [review]

File: app/scss/_footer.scss (hunk 1 above)
Comment (michaljedro) [reply app/scss/_footer.scss:22]: i forgeto to remove it...

File: app/scss/_footer.scss (hunk 2 above)
Comment (michaljedro) [reply app/scss/_footer.scss:25]: ok. i used $orangeColor fot it.

File: app/scss/_footer.scss (hunk 3 above)
Comment (michaljedro) [reply app/scss/_footer.scss:61]: done

File: app/scss/_footer.scss (hunk 4 above)
Comment (michaljedro) [reply app/scss/_footer.scss:65]: i remove at all that from project

Comment (michaljedro) [reply app/scss/_footer.scss:65]: i guess i can used for it just style background-color with $grayColor.

File: app/scss/_footer.scss (hunk 5 above)
Comment (michaljedro) [reply app/scss/_footer.scss:55]: ok. done

File: app/scss/_footer.scss (hunk 6 above)
Comment (michaljedro) [reply app/scss/_footer.scss:37]: ok. i changed it to header

File: app/scss/_footer.scss (hunk 8 above)
Comment (michaljedro) [reply app/scss/_footer.scss:51]: i know that method to select items in css

File: index.html (hunk 9 above)
Comment (michaljedro) [reply index.html]: i remove space

File: index.html (hunk 10 above)
Comment (michaljedro) [reply index.html:88]: i used it to positioned in element.

Comment (michaljedro) [reply index.html:88]: but it works without that too

Comment (michaljedro) [reply index.html:88]: I remove it

File: index.html (hunk 11 above)
Comment (michaljedro) [reply index.html:104]: ok. i remove it

File: index.html (hunk 12 above)
Comment (michaljedro) [reply index.html]: ok. i changed it.

File: app/scss/_colors.scss [hunk 13]
```diff
@@ -1 +1,3 @@
 $grayColor: #292f32;
+$orangeColor: #f09713;
```
Comment (pawel-schmidt) [inline app/scss/_colors.scss:2]: If client will ask you to change this color to green, you'll need to change the variable name. Better would be to name it like `primaryColor`. But let it be as is ;-)

**Review by pawel-schmidt — APPROVED** [review]

**Pull request merged**